{{cookiecutter.project_name}}/src/app.js

```diff
@@ -26,7 +29,7 @@ export default function createApp() {
 
     // enable routers
     express.use('/api', apiRouter);
-    express.use('/content', contentRouter);
+    express.use('/gql', contentRouter);
```

Comment:
probably call this gqlRouter if you are switching to `gql`. Also probably add the `health` endpoint here.